enterprise/server/cmd/ci_runner/main.go

```diff
@@ -413,21 +384,11 @@ func (r *buildEventReporter) Start(startTime time.Time) error {
 		startedEvent.Children = append(startedEvent.Children, &bespb.BuildEventId{Id: &bespb.BuildEventId_WorkflowConfigured{WorkflowConfigured: &bespb.BuildEventId_WorkflowConfiguredId{}}})
 		r.Printf("Streaming workflow logs to: %s", invocationURL(*invocationID))
 	} else {
-		startedEvent.Children = append(startedEvent.Children, &bespb.BuildEventId{Id: &bespb.BuildEventId_Pattern{Pattern: &bespb.BuildEventId_PatternExpandedId{Pattern: patterns}}})
 		r.Printf("Streaming remote runner logs to: %s", invocationURL(*invocationID))
 	}
 	if err := r.bep.Publish(startedEvent); err != nil {
 		return err
 	}
-	if !r.isWorkflow {
```

Comment:
If we're supporting multiple commands for remote bazel, it doesn't make sense for the outer invocation to emit this event. Unfortunately the UI will contain less information about the build (like the target that was run), even if only 1 bazel command is run

Reply:
wdyt about having a special case if there's only 1 command? (since that will probably be the most common case?) if it's hard because of the lossy bazel command => bash script conversion, maybe we can send up some extra command/pattern metadata to avoid losing this information?

Reply:
I thought about that, but the special case is still hard given the openness of bash scripts - 1 bash command could contain 3 bazel and 4 bash commands separated by && and ; 

But once we start processing the build logs for bazel invocations, we could start sending this again, and only using it to populate an invocation if there's only 1 bazel invocatoin

Reply:
I mean for stuff like `bb remote build //...` we only allow passing 1 command at the command line right? wonder if we'll lose the `build //...` information in this case, or if we could send up extra metadata to preserve that info

Reply:
I'd prefer not to add special logic around that, when we will already have a way to detect single bazel builds. But while we're in the intermediate phase of not processing the build logs yet, I can just re-add this code back in if you'd prefer

Reply:
What's the single way of detecting single builds that you're referring to?

Reply:
We'll have logic that's parsing the runner logs and looking for invocation links. We can use that to determine if there's just one bazel build

Reply:
We might have some difficulty with that approach because we want to publish the workflow invocation metadata as early as possible, and the metadata includes the command - but with the log scanning approach, we don't know whether there's more than one invocation until the second invocation has started

Reply:
I see. What if we added a name field to `Step`? I know we've wanted that anyway, so people could specify blocks like `Setup` and `Teardown`. Remote bazel could set the name to the bazel command

Reply:
That seems like a reasonable option if it winds up looking nice in the UI